tapgarden/mock.go

```diff
@@ -147,27 +148,47 @@ func FundGenesisTx(packet *psbt.Packet, feeRate chainfee.SatPerKWeight) {
 	packet.UnsignedTx.TxOut[1].Value -= int64(fee)
 }
 
+// FundPsbt funds a PSBT.
 func (m *MockWalletAnchor) FundPsbt(_ context.Context, packet *psbt.Packet,
-	_ uint32, feeRate chainfee.SatPerKWeight) (*tapsend.FundedPsbt, error) {
+	_ uint32, _ chainfee.SatPerKWeight,
```

Comment:
super nit: can split above function to reuse here.

Reply:
There's a lot of code duplication between `FundPsbt` and `FundGenesisTx`, which could be extracted. But non-blocking IMO.

Reply:
yeah I took a quick look and it wasn't very obvious to me, gonna skip for now as to not further delay merge